app/views/_mobile-service-instance-row.html

```diff
@@ -0,0 +1,90 @@
+<div class="list-pf-item provisioned-service mobile-service-instance-row" ng-class="{ active: row.expanded }">
+  <div class="list-pf-container" ng-click="row.toggleExpand($event)">
+    <div class="list-pf-chevron">
+      <div ng-include src=" 'views/overview/_list-row-chevron.html' " class="list-pf-content"></div>
+    </div>
+    <div class="list-pf-content">
+      <div class="list-pf-name">
+        <h3>
+          <span class="logo" ng-if="row.serviceClass.spec.externalMetadata.imageUrl">
+            <img src="{{row.serviceClass.spec.externalMetadata.imageUrl}}">
```

Comment:
`title` or `alt` on the `img` for accessibility purposes?

Reply:
:+1: